counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
+    if subasset_longname is not None:
+        cursor = db.cursor()
+        cursor.execute('''SELECT * FROM issuances \
+                          WHERE (status = ? AND asset = ?)
+                          ORDER BY tx_index ASC''', ('valid', subasset_parent))
```

Comment:
In isolation, but knowing how the issuance of subassets works, it's safe to assume a subasset has a parent. Right?